config/database.yml

```diff
@@ -20,8 +20,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  user: postgres
-  host: <%= ENV['POSTGRES_HOST'].present? ? ENV['POSTGRES_HOST'] : 'operationcode-psql' %>
+  user: <%= ENV['POSTGRES_USER'] %>
+  host: <%= ENV['POSTGRES_HOST'] %>
```

Comment:
What's the thought behind removing the default value?

Reply:
Default values are the names of docker containers which should be applied only when you want to run the environment in docker. Thus they should be optional and we can use the `.env` file to read the defaults when we use docker.